astroquery/esasky/tests/test_maps.py

```diff
@@ -0,0 +1,23 @@
+from astroquery.esasky import ESASky
+import astropy.units as u
+from astropy.coordinates import SkyCoord
```

Comment:
this is a good interactive test but the tests below should all be put into some function `def test_something` so that it's not called at import time and is run when we run overall tests